wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss

```diff
@@ -1,31 +1,175 @@
 .sensei {
-	&:not(.post-type-archive-course):not(.tax-course-category) {
-		section {
-			padding: 0;
-		}
-	}
+  &:not(.post-type-archive-course):not(.tax-course-category) {
+    section {
+      padding: 0;
+    }
+  }
 }
 
 .single-course {
-	.course-lessons {
-		.lesson {
-			border-bottom: none;
-			margin-bottom: 0;
-		}
-
-		.lesson-title {
-			h2 {
-				font-size: 1rem;
-				font-weight: 400;
-			}
-		}
-
-		.lesson-meta {
-			margin-bottom: 0;
-		}
-
-		.entry {
-			display: none;
-		}
-	}
+  .course-lessons {
+    .lesson {
+      border-bottom: none;
+      margin-bottom: 0;
+    }
+
+    .lesson-title {
+      h2 {
+        font-size: 1rem;
+        font-weight: 400;
+      }
+    }
+
+    .lesson-meta {
+      margin-bottom: 0;
+    }
+
+    .entry {
+      display: none;
+    }
+  }
+}
+
+.sensei {
+  .course {
```

Comment:
I think these nested selectors (`.course`, `.lesson`, `.quiz`) should be nested under the existing `.sensei` selector on line 1